addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1957,8 +1951,14 @@ const Wysiwyg = Widget.extend({
             button.classList.toggle('active', button.dataset.value === value);
         }
     },
-    _editorOptions: function () {
-        return Object.assign({}, this.defaultOptions, this.options);
+    _getEditorOptions: function (options) {
+        const finalOptions = {...this.defaultOptions, ...options};
+        // autohideToolbar is true by default (false by default if navbar present).
+        finalOptions.autohideToolbar = typeof finalOptions.autohideToolbar === 'boolean'
+            ? finalOptions.autohideToolbar
+            : !options.snippets;
```

Comment:
Isn't this functionaly equivalent to `finalOptions.autohideToolbar ??= !options.snippets;` ?

Reply:
This does not pass eslint.